### PR TITLE
Try to update vorta dependencies

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Merge as soon as Kubuntu 22.04 is out.
(Since mounting still doesn't work, I might have to EOL the Vorta flatpak)

### CODE REVIEW

File: dependencies/python3-fusermount.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                }
+            }
+        },
+        {
+            "type": "patch",
+            "path": "fuse-2.9.2-namespace-conflict-fix.patch"
+        },
+        {
+            "type": "file",
+            "path": "fusermount-wrapper.sh"
```
Comment (hfiguiere) [inline dependencies/python3-fusermount.json:29]: this file is missing from the PR :-(

File: dependencies/python3-fusermount.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                "project-id": 861,
+                "url-template": "https://github.com/libfuse/libfuse/releases/download/fuse-$version/fuse-$version.tar.gz",
+                "versions": {
+                    "<": "3"
+                }
+            }
+        },
+        {
+            "type": "patch",
+            "path": "fuse-2.9.2-namespace-conflict-fix.patch"
```
Comment (hfiguiere) [inline dependencies/python3-fusermount.json:25]: this patch is missing too

File: dependencies/python3-fusermount.json (hunk 1 above)
Comment (Hofer-Julian) [reply dependencies/python3-fusermount.json:29]: It was there before already

File: dependencies/python3-fusermount.json (hunk 2 above)
Comment (Hofer-Julian) [reply dependencies/python3-fusermount.json:25]: Same here

**Comment by hfiguiere** [comment]
bot, build

File: dependencies/python3-fusermount.json (hunk 2 above)
Comment (hfiguiere) [reply dependencies/python3-fusermount.json:25]: right, I missed it

**Comment by Hofer-Julian** [comment]
As mentioned above, even though I've got it to build, mounting doesn't work.

Plus there is a bug on Kubuntu 22.04 that causes that the directorychooser freezes the whole app.

**Comment by hfiguiere** [comment]
I just tried that build and it actually crashes. But since it upgrades PyQt as well. 

My PR does the minimum in changes. Even the upgrade of crypto was a failure.

**Pull request closed without merging**